aerogear/aerogear-android-sdk
# [WIP] Aerogear 2223 - parse mobile-services.json and add cert values to httpservice requests

## Motivation

JIRA: https://issues.jboss.org/browse/AEROGEAR-2223

## Description

We want the user to be able to easily configure the certificates to be used to perform requests, without the need to write code.

## Progress

- [x] Parse JSON
- [x] Pin Certs with okHttp
- [ ] JavaDoc

## To Verify
- Run Example App
- Navigate to Auth and click Authenticate
- Chrome Browser should boot and you should see keycloak log in
- Edit keycloak hash value in mobile-services.json file in the example app
- Restart Example App
- Navigate to Auth and click Authenticate
- Chrome Browser will not boot, and a `SSLPeerUnverifiedException` is logged
